### ISSUE TITLE
🐛 Bug Report: Wrong application log level is used for business logs

### ISSUE THREAD

**Issue by overbit** [amplication/amplication#5246]
### What happened?

`amplication-server` is currently logging the following error but this should not be of level `error` since it is representing the correct application execution.

```json
{"level":"error","message":"LimitationError: Allowed services per workspace: 3","service":"amplication-server","severity":"ERROR","stack":"Error: LimitationError: Allowed services per workspace: 3\n    at BillingService.<anonymous> (/app/main.js:1:498708)\n    at Generator.next (<anonymous>)\n    at fulfilled (/app/node_modules/tslib/tslib.js:164:62)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2023-02-21T11:07:50.764Z"}
```

### What you expected to happen

No logs or logs as `info` should be triggered instead when the situation happens.

### How to reproduce

On the client, created on free plan more then 3 services and check the logs from the server console.

### Amplication version

v1.3.0

**Comment by mahi072** [comment]
Hi @overbit, I would like to work on this issue.
Can you please direct me where exactly in the file I have to make changes, So that "log as info" should not be triggered? if possible ?

**Comment by souravjain540** [comment]
@mahi072 are you still interested in taking this issue?

**Comment by vinodsai-a** [comment]
@souravjain540 Hi, can I pick this up?

**Comment by overbit** [comment]
@vinodsai-a let us know if you need any support and good luck!

**Comment by vinodsai-a** [comment]
Hey @overbit, yes, can you help here? 
 https://discord.com/channels/757179260417867879/1084810149929156608/1084810149929156608

**Comment by souravjain540** [comment]
@vinodsai-a any updates?

**Comment by ognjenjevremovic** [comment]
There seams to be no activity on this issue, for quite some time now.  
May I take this issue and provide PR?

**Comment by ognjenjevremovic** [comment]
Does the same also applies to the `LimitationError: Allowed entities per service` log as well?  
I can see that the instance of `ValidationError` is thrown but this also seems like the correct application execution and not an actual error from an application standpoint.

**Comment by overbit** [comment]
@ognjenjevremovic yes you are correct. It applies to both

**Comment by ognjenjevremovic** [comment]
Thanks for the headsup @overbit!

Feel free to re-assign the issue to me.
I've included the changes in the PR #5966.
